dump/match_infra.py

```diff
@@ -250,6 +250,10 @@ def clear(self, namespace=None):
         elif namespace in self.cache:
             del self.cache[namespace]
 
+    def fill(self, ns, conn, connected_to):
```

Comment:
@vivekreddynv imho, looks like a hack. Can we have a better approach here?

Reply:
To construct the connection objects on the fly, we already have the logic in ConnectionPool. 
To fill the cache with pre-initialized objects I've added this function. 

The whole purpose of ConnectionPool is to maintain one SonicV2Connector Object per namespace and keep track of which tables it is already connected to. Given that, this methods suffices the requirement